### PR TITLE
[12.0][MIG] partner_communication_tag: migration to 12.0

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

File: partner_communication_tag/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    'summary': """
+    Let you add tags to a contact to manage your communication
+    strategies.
+    """,
+    'description': """
+    """,
+
+    'author': 'Coop IT Easy, Rémy Taymans',
+    'license': 'AGPL-3',
+    'version': '12.0.1.0',
```
Comment (vvrossem) [inline partner_communication_tag/__manifest__.py]: It should be [`ODOO_VERSION.MAJOR.MINOR.PATCH`](https://semver.org/)
=> `12.0.1.0.0`

File: partner_communication_tag/models/partner_communication_tag.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from odoo import models, fields
+
+
+class PartnerCommunicationTag(models.Model):
+    _name = 'res.partner.communication.tag'
+    _description = "Partner Communication Tag"
+    _order = 'name'
+
+    name = fields.Char("Name", required=True)
+    description = fields.Text("Description")
```
Comment (vvrossem) [inline partner_communication_tag/models/partner_communication_tag.py]: use `translate=True` to translate field values

**Review by vvrossem — CHANGES_REQUESTED** [review]
as pre-commit hook is not configured yet, paint it with black :art:

File: partner_communication_tag/models/partner_communication_tag.py (hunk 2 above)
Comment (huguesdk) [reply partner_communication_tag/models/partner_communication_tag.py]: should this be added for both `name` and `description` fields?

**Review by vvrossem — APPROVED** [review]

**Pull request merged**